core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -4421,9 +4594,9 @@ object PlaintextAdminIntegrationTest {
   }
 
   def checkInvalidAlterConfigs(
-    test: KafkaServerTestHarness,
-    admin: Admin
-  ): Unit = {
+                                test: KafkaServerTestHarness,
```

Comment:
Can you please revert these kinds of pure white-space changes from the PR?

In the Apache Kafka project, we typically try to avoid changing lines that we don't have to, because the change history (git blame) is often quite important to understand things.